ibm_mq/datadog_checks/ibm_mq/connection.py

```diff
@@ -26,27 +26,26 @@ def get_normal_connection(config):
     """
     Get the connection either with a username and password or without
     """
+    cd = _get_cd(config)
+    qmgr = pymqi.QueueManager(None)
```

Comment:
Seems like this matches the [pyqmi docs](https://dsuch.github.io/pymqi/examples.html), but the previous name was clearer:

```suggestion
    queue_manager = pymqi.QueueManager(None)
```